doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -788,8 +788,7 @@ OSSL_CMP_CTX_set0_trustedStore() was renamed to OSSL_CMP_CTX_set0_trusted(),
 using macros, while keeping the old names for backward compatibility,
 in OpenSSL 3.1.
 
-OSSL_CMP_CTX_get0_validatedSrvCert() and OSSL_CMP_CTX_reset_geninfo_ITAVs()
-were added in OpenSSL 3.1.
+OSSL_CMP_CTX_get0_validatedSrvCert() was added in OpenSSL 3.1.
```

Comment:
But there needs to be an entry for the added function saying it was added in OpenSSL 3.0.6

Reply:
Done as well.